Fluid/Values/FactoryValue.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+
+namespace Fluid.Values
+{
+    public class FactoryValue<T> : FluidValue
```

Comment:
Should we also add a static `Create` method to maintain consistency with other `FuildValue`s?

Reply:
Couldn't find any consistency so far, and that's ok like this,